crates/sovereign-sdk/full-node/sov-ledger-rpc/src/client.rs

```diff
@@ -70,7 +71,7 @@ pub trait Rpc {
     #[method(name = "getBatchProofsBySlotHeight")]
     async fn get_batch_proofs_by_slot_height(
         &self,
```

Comment:
there is a [u8; 32] in the above function we forgot to update, can you also update that?

also the (u64, u64) in get_soft_confirmation_range